src/jit/importer.cpp

```diff
@@ -7473,48 +7447,65 @@ GenTreePtr                Compiler::impFixupCallStructReturn(GenTreePtr     call
             compLongUsed = true;
         else if (((returnType == TYP_FLOAT) || (returnType == TYP_DOUBLE)) && (compFloatingPointUsed == false))
             compFloatingPointUsed = true;
+
+#if FEATURE_MULTIREG_RET
+        unsigned retRegCount = retTypeDesc->GetReturnRegCount();
+        assert(retRegCount != 0);   // This is current hit by ARM32 
```

Comment:
Yes, I need to remove this comment.
At one point in the past this assert was firing, but I subsequently fix this issue.